libkernelflinger/log.c

```diff
@@ -169,5 +187,100 @@ void log(const CHAR16 *fmt, ...)
 
 	va_start(args, fmt);
 	vlog(fmt, args);
+
+	if (KERNELFLINGER_LOG_OUTPUT) {
```

Comment:
**Coding style violation**: No bracket for a single instruction.

Reply:
Already modified.